package.json

```diff
@@ -91,6 +99,7 @@
         "bn.js": "^4.11.8",
         "bower": "1.8.8",
         "browserify": "^16.5.0",
+        "bundlesize": "^0.18.0",
```

Comment:
Couldn't you instead of installing it as ``devDependency`` also define the version range for the ``bundlesize`` package over the ``npx`` command? I also think ``npx`` isn't required here because we have added the ``bundlesize`` package as ``devDependency`` to the project.